src/controllers/userController.js

```diff
@@ -187,7 +182,7 @@ const createNewUser = async (req, res) => {
       password: pwd,
```

Comment:
returning password???

Reply:
I am not returning anything, the response message just says "user created"